openshift/templates/pims-api/build.yaml

```diff
@@ -50,6 +50,7 @@ objects:
           from:
             kind: ImageStreamTag
             name: ${DOTNET_BUILDER_IMAGE}:${DOTNET_BUILDER_TAG}
+          # scripts: https://raw.githubusercontent.com/bcgov/PIMS/dev/.s2i/bin
```

Comment:
This is required to resolve the user permission issue when attempting to run the `dotnet ef database update` command on deployment.  There are other issues that stop the hook from working however.